server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexCannedResponse.scala

```diff
@@ -59,6 +59,54 @@ trait AddressIndexCannedResponse {
     )
   }
 
+  def LimitNotNumeric: AddressBySearchResponseContainer = {
```

Comment:
All of these follow the same pattern, to reduce the lines here do something like this:

```
private def BadRequestTemplate(errors: AddressResponseError*): AddressBySearchResponseContainer = {
     AddressBySearchResponseContainer(
       response = Error,
       status = BadRequestAddressResponseStatus,
       errors = errors
     )
 }
```

then you can reduce each of these to:

```
def OffsetNotNumeric: AddressBySearchResponseContainer = {
     BadRequestTemplate(OffsetNotNumericAddressResponseError)
}
```